src/components/ebay-donut-chart/components/ebay-chart-legend/index.marko

```diff
@@ -0,0 +1,45 @@
+import type { DonutLegendItem } from "../../component";
+export interface Input {
+    focusIndex: number;
+    items: DonutLegendItem[];
+    "on-focus": (index: number) => void;
+    "on-blur": () => void;
+}
+class {
+    handleFocus(id: number) {
+        this.emit("focus", id);
+    }
+
+    handleBlur() {
+        this.emit("blur");
+    }
+}
+
+<div class="ebay-donut-chart__legend">
```

Comment:
We should maybe leverage an HTML description list here?

Reply:
I did update this to use ul/li

It gets a little bit tricky using `dl` `dt` `dd` when styling, I could be wrong, but I'm not sure it allows for other elements to be nested within `dl`